uint8 ControlType
geometry_msgs/Pose   CmdPose
geometry_msgs/Twist  CmdVel

uint8 POSITION_CONTROL_ONLY=1
uint8 VELOCITY_CONTROL_ONLY=0
